app/Http/Requests/Photo/SetPhotosTagsRequest.php

```diff
@@ -17,12 +17,17 @@ class SetPhotosTagsRequest extends BaseApiRequest implements HasPhotos, HasTags
 	use HasTagsTrait;
 	use AuthorizeCanEditPhotosTrait;
 
+	public const SHALL_OVERRIDE_ATTRIBUTE = 'shall_override';
```

Comment:
For the sake of consistency, I think a trait `HasShallOverrideTrait` should be created.

Reply:
I have no real opinion about that. I typically start creating a trait if at least three but at minimum two different classes use the same thing. `shall_override` is very unique to the `SetPhotosTagsRequest` (at least for the moment).

Personally, I would not create a trait for it. This only adds an additional step of indirection and complexity without saving any lines of code. But that is a question of personal taste.